Client/Frontend/Browser/SearchLoader.swift

```diff
@@ -24,14 +24,14 @@ class _SearchLoader<UnusedA, UnusedB>: Loader<Cursor<Site>, SearchViewController
     fileprivate let urlBar: URLBarView
     fileprivate let frecentHistory: FrecentHistory
 
-    private var isFirstQuery: Bool
+    private var skipNextAutocomplete: Bool
 
     init(profile: Profile, urlBar: URLBarView) {
         self.profile = profile
         self.urlBar = urlBar
         self.frecentHistory = profile.history.getFrecentHistory()
 
-        self.isFirstQuery = true
+        self.skipNextAutocomplete = false
```

Comment:
We don't want to skip the next (previously first) autocomplete automatically. We now explicitly specify when we want to skip now via `setQueryWithoutAutocomplete()`.

Reply:
Very nice!